spec/amp-actions-and-events.md

```diff
@@ -343,3 +342,7 @@ actions that apply to the whole document.
     <td>Updates <code>amp-bind</code>'s state. See <a href="../extensions/amp-bind/amp-bind.md#ampsetstate">details</a>.</td>
   </tr>
 </table>
+
+### `amp-access`
+
+`amp-access` target is a special target provided by the `amp-access` spec. See [details](../extensions/amp-access/amp-access.md#login-link).
```

Comment:
1) Would it be accurate and worthwhile to inform the reader what you observed? That is, that you can't give an arbitrary ID to this target.
2) Also worth mentioning: that actions for the `amp-access` target are dynamic depending on the structure
of the `amp-access` JSON.
3) What do you mean "provided by the spec"? Please be more specific on what provides amp-access.  Or alternatively, drop this and focus on why this target is special as mentioned in points 1 and 2.